presto-main/src/main/java/com/facebook/presto/metadata/StaticFunctionNamespace.java

```diff
@@ -685,7 +685,7 @@ public boolean isAggregationFunction(QualifiedName name)
         return Iterables.any(functions.get(name), function -> function.getSignature().getKind() == AGGREGATE);
     }
 
-    public FunctionMetadata getFunctionMetadata(FunctionHandle functionHandle)
+    public FunctionMetadata getFunctionMetadata(StaticFunctionHandle functionHandle)
```

Comment:
I assume in the future `FunctionNamespace` will have the same signature:
```java
public FunctionMetadata getFunctionMetadata(FunctionHandle functionHandle)
```
So maybe just use `FunctionHandle` rather than `StaticFunctionHandle`?